src/Sylius/Behat/Resources/config/suites/cli/canceling_unpaid_orders.yml

```diff
@@ -0,0 +1,29 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        cli_canceling_unpaid_orders:
+            contexts:
+                - sylius.behat.context.cli.cancel_unpaid_orders
+
+                - sylius.behat.context.domain.managing_orders
```

Comment:
IMHO it should be below transformers

Reply:
Well it's sorted alphabetically so I don't want to move anything.

Reply:
we are sorting it alphabetically but also we are separating it by type of context.
consistency in Sylius is the really important thing when you do something the first time I suggest checking a few different files instead of introducing your own new rules 🖖

Reply:
I'm not introducing my own rules... I wasn't aware of additional sorting rule